R/cluster_sce.R

```diff
@@ -0,0 +1,35 @@
+#' Perform clustering on a SingleCellExperiment object
+#'
+#' @param sce SingleCellExperiment object that requires clustering
+#' @param pca_name The name used to access the PCA results
+#' @param BLUSPARAM A BlusterParam object specifying the clustering algorithm to
+#' use
+#' @param cluster_column_name The name of the column to store the clustering
+#' results in the SCE; for naming you may want to include the type of clustering
+#' and k value of centers, e.g. `kmeans_10`
+#' @param seed Seed for reproducibility of clustering results
+#' @param ... Allows for additional arguments to be provided to `clusterRows()`
+#'
```

Comment:
We probably need this line in roxygen comments - 

```suggestion
#' @import SingleCellExperiment
```